etc/schema.json

```diff
@@ -1052,7 +1058,10 @@
                 },
                 "version": {
                     "description": "The version of this EVE DNS event",
-                    "type": "integer"
+                    "type": "integer",
+                    "suricata": {
+                        "keywords": false
```

Comment:
Is it good to have this `suricata.keyword` that can either be a boolean or an array ?

Reply:
I'm ok with it, unless its going into something the hardcodes a schema based on whats first seen., so not in eve.  Prevents conflict with another field as well..

```
"keywords": ["one", "two"],
"no-keywords": "true",
```

Reply:
Why do not we simply use the absence of `"keywords"` as an indication that there are no keywords ?

Reply:
Then empty array instead of false ?